### ISSUE TITLE
Model saving and data prediction

### ISSUE THREAD

**Issue by liouxy** [motefly/DeepGBM#5]
Hello,
I've noticed the train_DEEPGBM method will return three variables,
`return deepgbm_model, opt, metric`
but the main function didn't handle these outputs,
```
elif args.model == "deepgbm":
        num_data = dh.load_data(args.data+'_num')
        cate_data = dh.load_data(args.data+'_cate')
        # designed for faster cateNN
        cate_data = dh.trans_cate_data(cate_data)
        train_DEEPGBM(args, num_data, cate_data, plot_title)
```

Could you tell me how to save the model for inference after training? 
And I also want to know how to preprocess the data and how to feed the data to the saved model.  

Thank you very much!

**Comment by motefly** [comment]
The model is based on PyTorch, please consider saving them by some API of PyTorch. For LightGBM model, see the docs of it.
For preprocessing, please see https://github.com/motefly/DeepGBM/blob/master/experiments/preprocess/example.sh .
